PHPGraphQLArticle.md

```diff
@@ -131,17 +129,13 @@ So this schema has a lot to declare:
 * How to resolve that query. In the example, a service method (find) is called with the passed args
 * It also declares the Post type, with strong typed fields
 
-It's similar to an Swagger file, right? But, it's part of our system. Part of our development proccess is to declare that schema. This is one big improvement over REST: 
-
-*documentation is generated in the development process*
-
-Documentation generated in the process is for sure, more trustable regarding it's "uptodatiness". 
+It's similar to an Swagger file, right? But, here part of our development proccess is to declare that schema. This is a big improvement over REST: *Documentation generated within the process is far more reliable for 'up-to-dateness'.*
 
-Let's imagine now we need to access this same service from a mobile device to make a very simple and optimized list of all posts. We just need two fields, title and id. How would you do that in REST? Well, you probably would need to pass a parameter to specify this return type and code it inside your controller, putting a switch of if somewhere.
+Let's imagine now that we need to access this same service from a mobile device to make a simple list of all posts. We just need two fields: title and id. 
 
-In GraphQL, all you need is to change the query to say what fields you need. 
+How would we do that in REST? Well, we probably would need to pass a parameter to specify this return type and code it inside our controller. In GraphQL, all we need to update is the query to say what fields are needed. 
```

Comment:
... all we need is to update the query to say... or ... all we need is to update the query to specify...